src/aaudio/AudioStreamAAudio.cpp

```diff
@@ -390,4 +390,46 @@ Result AudioStreamAAudio::getTimestamp(clockid_t clockId,
     }
 }
 
+ErrorOrValue<double> AudioStreamAAudio::calculateLatencyMillis() {
+    AAudioStream *stream = mAAudioStream.load();
+    if (stream != nullptr) {
```

Comment:
I suggest doing an early bailout if the stream is null instead of having an extra indent for the main logic.

Reply:
Done